src/app/settings/admin/rule-groups/component.ts

```diff
@@ -0,0 +1,200 @@
+import {NotificationService} from '@core/services/notification';
+import {DatacenterService} from '@core/services/datacenter';
+import {MLAService} from '@core/services/mla';
+import {UserService} from '@core/services/user';
+import {ConfirmationDialogComponent} from '@shared/components/confirmation-dialog/component';
+import {RuleGroup, RuleGroupType, AdminRuleGroup} from '@shared/entity/mla';
+import {MLAUtils} from '@shared/utils/mla-utils';
+import {UserSettings} from '@shared/entity/settings';
+import _ from 'lodash';
+import {Subject} from 'rxjs';
+import {filter, switchMap, take, takeUntil} from 'rxjs/operators';
+import {Mode, AdminRuleGroupDialog} from './rule-group-dialog/component';
+
+@Component({
+  selector: 'km-admin-settings-rule-groups',
+  styleUrls: ['style.scss'],
+  templateUrl: 'template.html',
+})
+export class AdminSettingsRuleGroupsComponent implements OnInit, OnChanges, OnDestroy {
+  dataSource = new MatTableDataSource<AdminRuleGroup>();
+  displayedColumns: string[] = ['name', 'type', 'seed', 'actions'];
+  seeds: string[] = [];
+  seedFilter: string;
+  typeFilter: string;
+  settings: UserSettings;
+  ruleGroupTypes = Object.values(RuleGroupType);
+  adminRuleGroups: AdminRuleGroup[] = [];
+
+  @ViewChild(MatSort, {static: true}) sort: MatSort;
+  @ViewChild(MatPaginator, {static: true}) paginator: MatPaginator;
+
+  private _ruleGroupsMap = new Map<string, RuleGroup[]>();
+  private _unsubscribe = new Subject<void>();
+
+  constructor(
+    private readonly _matDialog: MatDialog,
+    private readonly _mlaService: MLAService,
+    private readonly _userService: UserService,
+    private readonly _notificationService: NotificationService,
+    private readonly _datacenterService: DatacenterService
+  ) {}
+
+  ngOnInit(): void {
+    this.dataSource.data = this.adminRuleGroups || [];
+    this.dataSource.sort = this.sort;
+    this.dataSource.paginator = this.paginator;
+    this.sort.active = 'name';
+    this.sort.direction = 'asc';
+
+    this._userService.currentUserSettings.pipe(takeUntil(this._unsubscribe)).subscribe(settings => {
+      this.settings = settings;
+      this.paginator.pageSize = settings.itemsPerPage;
+      this.dataSource.paginator = this.paginator; // Force refresh.
+    });
+
+    this._datacenterService.seeds.pipe(takeUntil(this._unsubscribe)).subscribe(seeds => {
+      this.seeds = seeds;
+      this._getAdminRuleGroups();
+    });
+  }
+
+  ngOnChanges(): void {
+    this.filter();
+  }
+
+  ngOnDestroy(): void {
+    this._unsubscribe.next();
+    this._unsubscribe.complete();
+  }
+
+  isPaginatorVisible(): boolean {
+    return (
+      this.adminRuleGroups &&
+      this.adminRuleGroups.length > 0 &&
+      this.paginator &&
+      this.adminRuleGroups.length > this.paginator.pageSize
+    );
+  }
+
+  hasNoData(): boolean {
+    return _.isEmpty(this.adminRuleGroups);
+  }
+
+  getName(data: string): string {
+    return MLAUtils.getRuleGroupName(data);
```

Comment:
This is a bit frightening, to be honest. In order to fill out a single name column, we will have to decode every string data object to YAML, transform it to JSON, cast it to object, and get the name.

This definitely requires an API change. I see that the `RuleGroup` CRD has a name that we should be using instead of this workaround.

![image](https://user-images.githubusercontent.com/2285385/148773596-6c05d62d-093d-40b7-9447-c4c88110a880.png)